vulnerabilities/migrations/0030_vulnerabilityserverity_compute_score.py

```diff
@@ -0,0 +1,24 @@
+from django.db import migrations
+
+from vulnerabilities.severity_systems import SCORING_SYSTEMS
+
+
+def compute_score(apps, schema_editor):
+    Vuln_severity = apps.get_model('vulnerabilities', 'VulnerabilitySeverity')
+    for vuln_severity in Vuln_severity.objects.all():
+        if vuln_severity.value and not vuln_severity.scoring_elements:
+            try:
+                vuln_severity.scoring_elements = SCORING_SYSTEMS[vuln_severity.scoring_system].as_score(vuln_severity.value)
```

Comment:
What if we compute a score that's not the same the one we had before?
We should find a way to do what I suggest as a possibility e.g, " 2.2 other way: group pairs of records and merge the two in one of the two records" and ensure that we do not end up computing a score that's not the score we have stored